stdlib/public/SwiftShims/RefCount.h

```diff
@@ -432,7 +431,14 @@ class RefCountBitsT {
   }
 
   LLVM_ATTRIBUTE_ALWAYS_INLINE
-  RefCountBitsT(RefCountBitsT<RefCountIsInline> newbits) {
+  RefCountBitsT(int dummy, RefCountBitsT<RefCountIsInline> newbits) {
```

Comment:
I tried this at first, and unfortunately it does not: MSVC get's confused, and starts complaining about ambiguous initializer lists - this happens anywhere where the dummy is the second parameter